cpp/src/groupby/groupby.cu

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2019-2020, NVIDIA CORPORATION.
+ * Copyright (c) 2019-2021, NVIDIA CORPORATION.
```

Comment:
no changes in this file. but copyright updated.

Reply:
Ah yes, I realized that this file has been changed in another recent merge.